op-challenger/game/fault/agent_test.go

```diff
@@ -170,7 +170,7 @@ func setupTestAgent(t *testing.T) (*Agent, *stubClaimLoader, *stubResponder) {
 	logger := testlog.Logger(t, log.LevelInfo)
 	claimLoader := &stubClaimLoader{}
 	depth := types.Depth(4)
-	gameDuration := 6 * time.Minute
+	gameDuration := 3 * time.Minute
```

Comment:
This was halved because a bit down stream from this, `NewAgent` now accepts the max clock duration. This variable name should be updated.

Reply:
The `gameDuration` is also used by the `Agent` to [determine](https://github.com/ethereum-optimism/optimism/blob/6f99544c57ef837afaa96f979a6154a4e8a3604a/op-challenger/game/fault/agent.go#L166) when to resolve subgames. We should tweak that logic to consider extended subgames. Otherwise, the challenger will be too eager when resolving claims - nbd, just annoying to look at in our logs.

For now, it's OK to add a TODO referencing an issue fixing `resolveClaims`. We can fix this later.

Reply:
I don't think I'd bother adding this complexity to the challenger. Once the challenger thinks the clock expires it does an `eth_call` to see if the claim will successfully resolve and only actually sends a transaction if that passes. So it won't even spam the logs [other than at trace level](https://github.com/ethereum-optimism/optimism/blob/6f99544c57ef837afaa96f979a6154a4e8a3604a/op-challenger/game/fault/agent.go#L182C4-L182C81).  Given this should be a very rare occurrence the extra eth_call seems fine.  We only added the clock check quite recently to reduce the amount of time the challenger spent making those eth_calls when there were lots of claims and a relatively slow RPC.